browser/modules/AboutNewTab.jsm

```diff
@@ -65,8 +68,12 @@ let AboutNewTab = {
    *          history tiles feature.
    */
   customize: function(message) {
-    NewTabUtils.allPages.enabled = message.data.enabled;
-    NewTabUtils.allPages.enhanced = message.data.enhanced;
+    if (message.data.enabled !== undefined) {
```

Comment:
`message.data.enabled` is a boolean so if we do a "truthy" check we will get enter the if-statement based on the value of `message.data.enabled` rather than based on whether it was defined or not.